lte/protos/mconfig/mconfigs.proto

```diff
@@ -530,3 +530,12 @@ message AgwD {
   // disabled.
   string sentry_dsn = 6;
 }
+
+//------------------------------------------------------------------------------
+// SctpD configs
+//------------------------------------------------------------------------------
+message SctpD {
+  orc8r.LogLevel log_level = 1;
+  // sentry_config stores the Sentry.io configuration for this service
+  SentryConfig sentry_config = 2;
```

Comment:
Instead of taking this value here, let's take it from the Shared mconfig

Reply:
I think it's still useful to have a Magma SCTPD config section though. (We probably have other configurations that should be changeable from the cloud)  But let's pull sentry config from shared mconfig.

Reply:
Done.